command/meta.go

```diff
@@ -483,6 +499,15 @@ func (m *Meta) outputShadowError(err error, output bool) bool {
 	return true
 }
 
+func (m *Meta) EnvPath() string {
+  envName := m.Workspace()
+	if envName == backend.DefaultStateName {
+		return ""
+	} else {
+		return filepath.Join(local.stateWorkspaceDir(), envName)
```

Comment:
Would this be the right place to look for these variable files? My understanding is that this directory would only exist when using local state?

Could this look for the variable files alongside the .tf files but within folders named after the workspace?